test/integration/crud/bulk.test.ts

```diff
@@ -1216,34 +1216,27 @@ describe('Bulk', function () {
     }
   });
 
-  it('should correctly execute unordered batch using w:0', {
-    metadata: { requires: { topology: ['single', 'replicaset', 'ssl', 'heap', 'wiredtiger'] } },
+  it('should correctly execute unordered batch using w:0', async function () {
```

Comment:
This test flaked locally, I refactored to async-await and it didn't fail again.  I don't know if the test was actually flaky, but I left the test refactored.